analytics_data_api/v0/models.py

```diff
@@ -230,18 +253,32 @@ def get_users_in_course(
             course_id,
             segments=None,
             ignore_segments=None,
-            # TODO: enable during https://openedx.atlassian.net/browse/AN-6319
-            # cohort=None,
+            cohort=None,
             enrollment_mode=None,
             text_search=None,
-            order_by='username',
-            sort_order='asc'
+            sort_policies=None,
     ):
         """
         Construct a search query for all users in `course_id` and return
-        the Search object.  Raises `ValueError` if both `segments` and
-        `ignore_segments` are provided.
+        the Search object.
+
+        sort_policies is an array, where the first element is the primary sort.
+        Elements in the array are dicts with fields: order_by (field to sort by)
+        and sort_order (either 'asc' or 'desc').  Default to 'username' and 'asc'.
+
+        Raises `ValueError` if both `segments` and `ignore_segments` are provided.
         """
+
+        if sort_policies is None or len(sort_policies) == 0:
+            sort_policies = [{
+                'order_by': None,
+                'sort_order': None
+            }]
+        # set default sort policy to 'username' and 'asc'
+        for field, default in [('order_by', 'username'), ('sort_order', 'asc')]:
+            if sort_policies[0][field] is None:
+                sort_policies[0][field] = default
```

Comment:
Dang! I thought that particular quirk only applied to the empty list, but I guess not :(. Oh well!